src/test/pow_tests.cpp

```diff
@@ -32,7 +34,9 @@ BOOST_AUTO_TEST_CASE(get_next_work_pow_limit)
     pindexLast.nHeight = 2015;
     pindexLast.nTime = 1233061996;  // Block #2015
     pindexLast.nBits = 0x1d00ffff;
-    BOOST_CHECK_EQUAL(CalculateNextWorkRequired(&pindexLast, nLastRetargetTime, chainParams->GetConsensus()), 0x1d00ffffU);
+    unsigned int expected_nbits = 0x1d00ffffU;
```

Comment:
can this just be the (casted) value of pindexLast.nBits?

Reply:
Well, it could be, but the test would still be correct if `pindexLast.nBits` were modified to be within a factor of 4 from the max difficulty target.

So it seems to me that having `expected_nbts = 0x1d00ffffU` is the more important line as it captures exactly what the test case is trying to exercise here (and `pindexLast.nBits` could be set from that, or not).